.github/workflows/e2e-cli.yaml

```diff
@@ -17,6 +17,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Shouldn't this be `1` to fetch only the last commit? (not sure but maybe 0 means everything)

Reply:
okay